libs/messenger-bundle/composer.json

```diff
@@ -20,18 +20,11 @@
             "Keboola\\MessengerBundle\\Tests\\": "tests/"
         }
     },
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "git@github.com:keboola/MessengerAzureBundle.git"
-        }
-    ],
-
     "require": {
         "php": ">=8.2",
         "ext-pcntl": "*",
-        "aymdev/messenger-azure-bundle": "dev-dsnParser",
-        "petitpress/gps-messenger-bundle": "^1.6",
+        "aymdev/messenger-azure-bundle": "^2.0",
```

Comment:
Pepovo změny jsou zahrnutý ve verzi 2 a neměl by být potřeba fork
https://github.com/AymDev/MessengerAzureBundle/pulls?q=is%3Apr+is%3Aclosed+author%3Apepamartinec